plugins/package-managers/maven/src/main/kotlin/Maven.kt

```diff
@@ -172,6 +172,10 @@ class Maven(
 
         return listOf(ProjectAnalyzerResult(project, emptySet(), issues))
     }
+
+    override fun afterResolution(definitionFiles: List<File>) {
```

Comment:
commit: Is this commit the primary reason for modifying the hooks in previous changes?

I wonder if introducing some `releaseResourcesHook()`, called before process termination would be a better approach for that. This could be called be the CLI commands.